bookmarks/services/importer.py

```diff
@@ -169,6 +172,18 @@ def _import_batch(
             logging.exception("Error importing bookmark: " + shortened_bookmark_tag_str)
             result.failed = result.failed + 1
 
+    if options.get_metadata_flag:
+        bookmarks_missing_metadata = filter(
+            lambda b: not b.title or not b.description,
+            bookmarks_to_create + bookmarks_to_update,
+        )
+
+        with ThreadPoolExecutor(5) as executor:
```

Comment:
This should be a background task instead, you can check `bookmarks/services/tasks.py` as a reference for existing ones. So you could add a new task there, then schedule one per bookmark from here.

Tests would be welcome as well, `test_bookmarks_tasks.py` is used for testing tasks themselves, and `test_importer.py` could test that the task is called on import.